packages/components/src/border-control/border-control-dropdown/hook.ts

```diff
@@ -64,18 +66,26 @@ export function useBorderControlDropdown(
 		return cx( styles.colorIndicatorWrapper( border ) );
 	}, [ border, cx ] );
 
+	// We only want to apply adjusted popover content styles when on desktop.
+	const responsiveSide = useResponsiveValue( [
+		undefined,
+		undefined,
+		__experimentalSide,
+	] );
```

Comment:
I couldn't find the `@include break-medium()` equivalent using Emotion within the components package. This was my best guess but very happy to switch this out for any better ideas.